meson.build

```diff
@@ -406,6 +404,9 @@ endif
 man_pages = []
 
 # Tools
+have_getopt_long = cc.has_header_symbol('getopt.h', 'getopt_long',
+                                        prefix: '#define _GNU_SOURCE',
+                                        required: get_option('enable-tools'))
```

Comment:
I find this passing a `feature` to `required` business pretty confusing. I prefer not using that...

Also, we need to preserve the behavior that by default if `getopt_long` is available, the tools are built, regardless of meson's `auto-features` flag. So I'm thinking we shouldn't use `type: feature` for the option, but keep it a bool with default true.

Maybe some day we'll add proper `feature` options and deprecate the `enable` options, but until then, better to be consistent.